src/_shared/scss/_core.scss

```diff
@@ -240,10 +251,47 @@ $font-scale: (
     line-height: convert-to-px($line-height);
 }
 
+// Hide content visually, still readable by screen readers
+@mixin u-h {
```

Comment:
You don't need this. If you do, please just add the class `u-h` to the element you want to hide.